### ISSUE TITLE
CLAM checkpoint problem on google drive

### ISSUE THREAD

**Issue by chickeyton** [mahmoodlab/CLAM#110]
https://drive.google.com/drive/folders/1NZ82z0U_cexP6zkx1mRk-QeJyKWk4Q7z

I tried to load the checkpoint
/camelyon_40x_cv_CLAM_10_s1/[s_0_checkpoint.pt](http://s_0_checkpoint.pt/) with code:

```
model = CLAM_MB(n_classes=2)

ckpt = torch.load(ckpt_pat)
ckpt_clean = {}
for key in ckpt.keys():
    if 'instance_loss_fn' in key:
        continue
    ckpt_clean.update({key.replace('.module',''):ckpt[key]})

model.load_state_dict(ckpt_clean, strict=True)

```
the code was copied from eval_utils.py, but there are errors in load_state_dict()
missing ckpt keys:  attention_net.2.attentionXXX ...
unexpected ckpt keys: attention_net.3.attentionXXX ...

Is there any way to load the ckpt properly ?

**Comment by chickeyton** [comment]
it should be model = CLAM_MB(dropout=True, n_classes=2)

**Comment by fedshyvana** [comment]
yeah you should enable dropout, otherwise you won't not get the same behavior that the checkpoint is intended for since we use dropout during training.
